lsr_fingerprint.py

```diff
@@ -44,12 +47,40 @@
                         ]
                         for tmpl in tmpls:
                             tmplpath = join(dirpath, tmpl)
-                            with open(tmplpath) as fp:
-                                lines = fp.read()
-                            newlines = lines.replace(
-                                "{0}:{1}".format(role["reponame"], role["rolename"]),
-                                "system_role:{0}".format(role["lsrrolename"]),
-                            )
-                            if lines != newlines:
-                                with open(tmplpath, "w") as fp:
-                                    fp.write(newlines)
+                            with open(tmplpath) as ifp:
+                                lines = ifp.read()
+                            if lines.find(oldpair):
+                                with open(tmplpath) as ifp:
+                                    lines = ifp.readlines()
+                                count = 0
+                                newlines = ""
+                                changed = {}
+                                for line in lines:
+                                    newline = line.replace(oldpair, newpair)
+                                    if newline != line:
+                                        if len(changed) > 0:
+                                            raise Exception(
```

Comment:
This will exit the script without processing any other lines or files - so you won't know how many other lines or files have more than 1 `oldpair` in them